internal/orchestrator/orchestrator_test.go

```diff
@@ -83,6 +84,33 @@ type testOrchestrator struct {
 	msd *shareddownloadmocks.Manager
 	mae *admineventsmocks.Manager
 	mdh *definitionsmocks.DefinitionHandler
+	mns *namespacemocks.Manager
+}
+
+func (tor *testOrchestrator) cleanup(t *testing.T) {
+	tor.mdi.AssertExpectations(t)
```

Comment:
That's a lot of sub-components :)

... maybe the namespaces work (in a future PR) is the time to look at whether all of this attach at the same layer, or if some attach at the `orchestrator` layer, but most attach one level lower at a new namespace-specific orchestration layer tier.

Reply:
I think it's already clear that the new `Namespace Manager` is going to shoulder a large part of orchestrator's responsibility by the time we're done. The final component tree is still foggy to me, but it has to end up a little better =)